packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -345,27 +312,16 @@ async function storeMrSummaryAndBriefInS3({
     });
   };
 
-  const promiseBriefSummary = async function () {
-    if (!aiBrief) return;
-    return newS3Client.uploadFile({
-      bucket: bucketName,
-      key: briefFileName,
-      file: Buffer.from(aiBrief),
-      contentType: "text/plain",
-    });
-  };
-
-  const [mrResp, briefResp] = await Promise.allSettled([promiseMrSummary(), promiseBriefSummary()]);
-  if (mrResp.status === "rejected" || briefResp?.status === "rejected") {
-    const failed = [mrResp, briefResp].map(p => (p.status === "rejected" ? p.reason : []));
-    const message = "Failed to store MR Summary and/or Brief in S3";
-    const additionalInfo = { reason: failed.join("; "), bucketName, htmlFileName, briefFileName };
+  const mrResp = await promiseMrSummary();
```

Comment:
If we're not using `Promise.all` anymore, we could simplify the logic and have it all inline.